portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,43 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /* List of fun facts */
+  private List<String> facts;
+  /* Pointer to the current index of facts given when the servlet is called */
+  private int currQuestionIndex;
+
+  /* Initializes facts list and currQuestionIndex */
+  @Override
+  public void init() {
+    currQuestionIndex = 0;
+    facts = new ArrayList<>();
+    facts.add("Some of my friends call me by the nickname \"Puffball\" due to "
+              + "my hair's tendency to get poofy.");
+    facts.add("I'm confident that I've played at least 1 video game from every "
+              + "Nintendo video game series.");
+    facts.add("Despite being Filipino, I can barely speak any. :( "
+              + "Trying to practice though so that one day...!");
+    facts.add("Although I listen to Pop most of the time, my guilty pleasure "
+               + "music are video game OSTs. Yoko Shimomura is just too good "
+               + "of a composer.");
+    facts.add("One of the career paths I once considered was being a writer.");
+    
+    Collections.shuffle(facts);
```

Comment:
Is this necessary?

Reply:
Yeah, I want fun facts to be delivered randomly but without repeats. When all the fun facts have been seen it'll go through them again but in a new order, so I use Collections.shuffle to do that.